src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -344,6 +350,9 @@
   <data name="Nav_Rendering.Content" xml:space="preserve">
     <value>Rendering</value>
   </data>
+  <data name="Nav_Actions.Content" xml:space="preserve">
+    <value>Keybindings</value>
+  </data>

```

Comment:
I'm torn on this now. I like Keyboard because it clearly shows the user where to find key bindings. However, our JSON uses Actions, so if someone wanted to edit their keys, would they know they'd have to edit the Actions array?

Reply:
Okay I'm going with Actions for now. The later in the day we get, the harder it will be to change again